src/NHibernate.Test/NHSpecificTest/Properties/Mappings.hbm.xml

```diff
@@ -27,6 +27,8 @@
       </key>
       <one-to-many class="Account"/>
     </set>
+    <!-- Having it mapped after the collection referencing it can cause trouble when resolving the collection, if
+         not accounted for. So keep it mapped after for checking this case is still supported. -->
```

Comment:
This pre-existing test is already checking the case of a collection having a property-ref on a "component like" (here a `properties` mapping) property mapped after the collection.